libcontainer/factory_linux.go

```diff
@@ -35,6 +35,14 @@ var (
 // provided init binary path and arguments.
 func InitArgs(args ...string) func(*LinuxFactory) error {
 	return func(l *LinuxFactory) error {
+		if len(l.InitArgs) > 0 {
```

Comment:
Again, I'm confused -- why isn't this `len(args)`? `l.InitArgs` hasn't been set yet.